cohesive_energy.py

```diff
@@ -0,0 +1,28 @@
+from ase.md.velocitydistribution import (MaxwellBoltzmannDistribution,Stationary,ZeroRotation)
+from ase.md.verlet import VelocityVerlet
+from asap3 import Trajectory
+from ase import units
+from equilibriumCondition import equilibiriumCheck
+
+
+
+def cohesive_energy(atoms,iterations):
+    """The cohesive_energy function takes argument atoms, that is the simulated material,
+    and makes another simulation for the material at temperature 0 K.
+    It the looks extracts the potential energy which is the cohesive energy for the material"""
+    MaxwellBoltzmannDistribution(atoms, temperature_K=0)
```

Comment:
Pass: communicator='serial' as parameter to Maxwell in order to enable multi process simulation.

Reply:
Fixed!